src/staticanalysis/bot/static_analysis_bot/coverity/coverity.py

```diff
@@ -35,6 +35,15 @@
 ```
 '''
 
+ISSUE_ELEMENT_IN_STACK = '''
+- **{file_path}:{line_number}**:
```

Comment:
Nit: Let's make it italic, with one *, otherwise it's a punch in the eye :P

Reply:
Sure!